lib/src/widgets/country_picker/country_selector.dart

```diff
@@ -25,17 +63,62 @@ class CountrySelector extends StatefulWidget {
 
 class _CountrySelectorState extends State<CountrySelector> {
   late List<Country> _filteredCountries;
-  late final _countryFinder;
+  late CountryFinder _countryFinder;
+  int? _separatorIndex;
```

Comment:
this is ambiguous as per the comment above, I'd rename it to _favoritesSeparatorIndex. It's ambiguous in the case we add another separator in the future for example a separator between each countries (with no favorites).

Reply:
updated